src/features/error/ErrorPage.js

```diff
@@ -1,37 +1,61 @@
-import PropTypes from "prop-types";
 import React from "react";
-import { useNavigate } from "react-router-dom";
+import { useNavigate, useLocation } from "react-router-dom";
 import styled from "styled-components";
 
+import errorCrying from "../../assets/error-crying.png";
 import ButtonSmall from "../../common/components/buttons/ButtonSmall";
 
 const StyledErrorPage = styled.div`
   width: 100%;
   height: 100%;
   background-color: black;
+  color: white;
   display: flex;
   flex-direction: column;
   align-items: center;
+  justify-content: center;
   gap: 1rem;
+  img {
+    width: 300px;
+  }
+  .error {
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    .error-title {
+      font-size: x-large;
+    }
+    .error-message {
+      color: gray;
+    }
+  }
 `;
 
-function ErrorPage({ error }) {
+function ErrorPage() {
   const navigate = useNavigate();
+  const location = useLocation();
+  const error = location.state;
 
   return (
     <StyledErrorPage>
-      <div className="title">{error.title}</div>
-      <div className="description">{error.description}</div>
+      <div className="image">
+        <img src={errorCrying} alt="crying baby" />
+      </div>
+      <div className="error">
+        <div className="error-title">
+          {error ? error.title : "{error.title}"}
```

Comment:
에러 페이지로 올때는 항상 location을 사용해서 오기때문에. 전달받은 error 객체가 있을거 같아요. 만약 개발 단계에서 화면을 보기위해 넣어놓은 로직이라면 실제 배포 버전에서는 바꿔야할 거 같네요. 
```suggestion
          {error.title}
```